src/pages/ReplaceServer/InstallOS.js

```diff
@@ -69,7 +74,7 @@ class InstallOS extends BaseUpdateWizardPage {
       name: INSTALL_PLAYBOOK,
       payload: {
         'extra-vars': {
-          'nodelist': [this.props.operationProps.server.id],
+          'nodelist': this.props.operationProps.server.id,
```

Comment:
this is a little confusing given the variable name, but I suppose its not really any different than before